### PR TITLE
monitoring: Only monitor site level external services error rates

### PR DESCRIPTION
**Pull request by ryanslade**
Tweak monitoring to only consider site level external services since we have no
control over how user added external services are configured.

### CODE REVIEW

File: monitoring/definitions/repo_updater.go [hunk 1]
```diff
@@ -60,13 +60,13 @@ func RepoUpdater() *monitoring.Container {
 						},
 						{
 							Name:        "src_repoupdater_syncer_sync_errors_total",
-							Description: "sync error rate",
-							Query:       `max by (family) (rate(src_repoupdater_syncer_sync_errors_total[5m]))`,
+							Description: "site level external service sync error rate",
+							Query:       `max by (family) (rate(src_repoupdater_syncer_sync_errors_total{owner!="user"}[5m]))`,
```
Comment (ryanslade) [inline monitoring/definitions/repo_updater.go:64]: This is the core change, just the `{owner!="user"}`

File: monitoring/definitions/repo_updater.go [hunk 2]
```diff
@@ -79,7 +79,7 @@ func RepoUpdater() *monitoring.Container {
 					{
 						{
 							Name:              "syncer_sync_start",
-							Description:       "sync was started",
+							Description:       "repo metadata sync was started",
 							Query:             `max by (family) (rate(src_repoupdater_syncer_start_sync[5m]))`,
```
Comment (indradhanush) [inline monitoring/definitions/repo_updater.go:83]: Do we want to group this alert also by `owner`? :thinking:

**Review by indradhanush — APPROVED** [review]
LGTM. Approve to unblock.

File: monitoring/definitions/repo_updater.go (hunk 2 above)
Comment (ryanslade) [reply monitoring/definitions/repo_updater.go:83]: Yep, that makes sense

**Pull request merged**